jenkins/integ-test-notification.jenkinsfile

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+lib = library(identifier: 'jenkins@7.0.0', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+
+pipeline {
+    agent { label 'Jenkins-Agent-Ubuntu2004-X64-M58xlarge-Single-Host' }
```

Comment:
Should not use this as this is specifically for gradle check and only runs once and delete.

Reply:
Do you have any recommendation?

Reply:
If you want to use ubuntu then use the docker builder one
If not, then use a generic x64 docker host.

Reply:
Updated to `Jenkins-Agent-AL2023-X64-C54xlarge-Docker-Host` 
Thanks!

Reply:
So you are going to run directly on agent node without docker container?

Reply:
Yes! Followed the same patter we are using for https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/gradle/gradle-check-flaky-test-issue-creation.jenkinsfile

Reply:
Adding @prudhvigodithi to take a look to update that jenkinsfile to use other runners.
I still think using a docker container here is more clean.
As using the agent might have cleanup issues.

Reply:
Used `Jenkins-Agent-Ubuntu2004-X64-M58xlarge-Single-Host` in jenkinsfile https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/gradle/gradle-check-flaky-test-issue-creation.jenkinsfile because:

- This `gradle-check-flaky-test-issue-creation.jenkinsfile` comes under the same family gradle-check hence used the same.
- Mainly using `Jenkins-Agent-Ubuntu2004-X64-M58xlarge-Single-Host` because of the latest `curl` version which supports `--aws-sigv4` flag.

But looks to me like this `Jenkins-Agent-AL2023-X64-C54xlarge-Docker-Host` (coming from https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/distribution-build.jenkinsfile#L876-L881) also has that latest `curl` version that supports `--aws-sigv4` flag.